docs/features.md

```diff
@@ -2,61 +2,108 @@
-- `vanderwaals`: Lennard-Jones potentials, given interatomic distance/s and a list of atoms with vanderwaals parameters. There's no distance cutoff here. The radius of influence is assumed to infinite. Float value.
+## Default node features 
+For atomic graphs, when features relate to residues then _all_ atoms of one residue receive the feature value for that residue.
+
+### Core properties of atoms and residues: `deeprankcore.features.components`
+These features relate to the chemical components (atoms and amino acid residues) of which the graph is composed. Detailed information and descrepancies between sources are described can be found in `deeprankcore.domain.aminoacidlist.py`.
+
+#### Atom properties:
+These features are only used in atomic graphs.
+
+- `atom_type`: One hot encoding of the atomic element. Options are: C, O, N, S, P, H.
+- `atom_charge`: Atomic charge in Coulomb (float). Taken from `deeprankcore.domain.forcefield.patch.top`.
+- `pdb_occupancy`: Proportion of structures where the atom was detected at this position (float). In some cases a single atom was detected at different positions, in which case separate structures exist whose occupancies sum to 1. Only the highest occupancy atom is used by deeprankcore. 
+
+#### Residue properties:
+- `res_type`: One hot encoding of the amino acid residue (size 20).
+- `polarity`: One hot encoding of the polarity of the amino acid (options: NONPOLAR, POLAR, NEGATIVE, POSITIVE). Note that sources vary on the polarity for few of the amino acids; see detailed information in `deeprankcore.domain.aminoacidlist.py`.
+- `res_size`: The number of non-hydrogen atoms in the side chain (int). 
+- `res_mass`: The (average) residue mass in Da (float).
+- `res_charge`: The charge of the residue (in fully protonated state) in Coulomb (int). Charge is calculated from summing all atoms in the residue, which results in a charge of 0 for all polar and nonpolar residues, +1 for positive residues and -1 for negative residues.
+- `res_pI`: The isolectric point, i.e. the pH at which the molecule has no net electric charge (float).
+
+- `hb_donors`, `hb_acceptors`: The number of hydrogen bond donor/acceptor atoms in the residue (int). Hydrogen bonds are noncovalent intermolecular interactions formed between an hydrogen atom (partially positively charged) bound to a small, highly electronegative atom (O, N, F) with an unshared electron pair.
+
+#### Properties related to variant residues:
+These features are only used in SingleResidueVariant queries.
+
+- `variant_res`: One hot encoding of variant amino acid (size 20).
+- `diff_charge`, `diff_polarity`, `diff_size`, `diff_mass`, `diff_pI`, `diff_hb_donors`, `diff_hb_acceptors`: Subtraction of the wildtype value of indicated feature from the variant value. For example, if the variant has 4 hb_donors and the wildtype has 5, then `diff_hb_donors == -1`.
```

Comment:
```suggestion
- `diff_charge`, `diff_polarity`, `diff_size`, `diff_mass`, `diff_pI`, `diff_hb_donors`, `diff_hb_acceptors`: Subtraction of the wildtype value of indicated feature from the variant value (int). For example, if the variant has 4 hb_donors and the wildtype has 5, then `diff_hb_donors == -1`.
```

Reply:
these are not all ints (e.g. charge is a float, polarity is a list of -1/0/1). They are all of the same type as the original. I would leave the explanation of that out, though, or do you think it's needed?

Reply:
Oh right sorry, no I don't think it's needed to change anything. It's pretty obvious that the type will be the same as the original feature.